src/Compilers/Core/Portable/Compilation/OptimizationLevel.cs

```diff
@@ -48,42 +48,46 @@ public enum OptimizationLevel
 
     internal static class OptimizationLevelFacts
     {
+        internal static (OptimizationLevel OptimizationLevel, bool DebugPlus) DefaultValues { get; } = (OptimizationLevel.Debug, false);
```

Comment:
>{ get; } = [](start = 92, length = 10)

It looks like property has a backing field. Is that intentional? If not, consider using `=>` instead.